examples/http/rest.cpp

```diff
@@ -27,13 +27,17 @@ static constexpr uint16_t default_port = 8080;
 
 static constexpr size_t default_max_connections = 128;
 
+static constexpr size_t default_max_request_size
+  = caf::net::http::server::default_max_request_size;
```

Comment:
If we want to expose this to users, it should go to `defaults.hpp` as `caf::defaults::net::http_max_request_size`.